Makefile

```diff
@@ -38,19 +38,19 @@ GO_DIR := $(OUTPUT_DIR)/go
 
 # Base image used for all golang containers
 # Uses trusted google-built golang image
-GOLANG_IMAGE_VERSION := 1.23.4
+GOLANG_IMAGE_VERSION := 1.24.2
```

Comment:
Are you planning to update the Go version to 1.24?

Reply:
let me see how much churn that introduces.

Reply:
Playbook is here: go/cs-upgrade-go (applicable if you're upgrading minor versions)

Reply:
thank you.  added the links for discoverability.  I think we can consider moving this playbook closer to the source of changes.

Reply:
Note it involves internal specific steps. I don't think it belongs in OSS.